lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -52,7 +52,13 @@ namespace Cpp {
     // Behaviour is to not add paths that don't exist...In an interpreted env
     // does this make sense? Path could pop into existance at any time.
     for (const char* Var : kSysLibraryEnv) {
+#if defined(_WIN32)
```

Comment:
Why do we need this block, getenv should be implementation independent?

Reply:
Because we get the warning described in this stack overflow. Given we have set all warnings as errors, we need to make the change.